.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSIBLE INFORMATION ABOUT YOUR
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Thank you for considering giving feedback to the project!
```

Comment:
2 cents

Repository Mirroring Tool (RMT) is an open-source project and we can make it even better with your help and feedback.